cloud_governance/policy/aws/zombie_cluster_resource.py

```diff
@@ -200,7 +200,7 @@ def zombie_cluster_volume(self, vpc_id: str = '', cluster_tag_vpc: str = ''):
                 if delete_cluster_resource and self.delete:
                     self.delete_ec2_resource.delete_zombie_resource(resource_id=zombie, resource='ec2_volume')
                 else:
-                    if self._force_delete:
+                    if self._force_delete and self.delete:
```

Comment:
Why we need self.delete in addition to self._force_delete ?

Reply:
Double verrification

Reply:
ok